### PR TITLE
cadical 1.4.0 (new formula)

### PR DESCRIPTION
**Pull request by 4tXJ7f**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by carlocab — COMMENTED** [review]
Thanks, @4tXJ7f. Just a few questions.

File: Formula/cadical.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  depends_on "make" => :build
+
+  def install
+    system "./configure"
+    chdir "build" do
+      system "gmake"
+      bin.install "cadical"
+      lib.install "libcadical.a"
+      include.install "../src/cadical.hpp"
```
[the remark this answers is not in the document]
Comment (4tXJ7f) [reply Formula/cadical.rb:14]: No, unfortunately, it doesn't. The generated makefile is based on [this](https://github.com/arminbiere/cadical/blob/master/makefile.in) template.

File: Formula/cadical.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  homepage "http://fmv.jku.at/cadical/"
+  url "https://github.com/arminbiere/cadical/archive/refs/tags/rel-1.4.0.tar.gz"
+  sha256 "b5990bea7aa5df6002b9bfe782323742d786565b9dfee37bbc0dd0f4b8e65141"
+  license "MIT"
+
+  def install
+    system "./configure"
+    chdir "build" do
+      system "make"
+      bin.install "cadical"
```
Comment (carlocab) [inline Formula/cadical.rb:12]: Should we not install `mobical`?

https://github.com/arminbiere/cadical/blob/master/BUILD.md#cadical-build

Comment (carlocab) [reply Formula/cadical.rb:12]: If/when you make further changes, squash them with your first commit, please. Thanks!

Comment (4tXJ7f) [reply Formula/cadical.rb:12]: I don't think that it makes sense to install `mobical`. It is tool for generating test cases for `cadical`'s API (see description [here](https://people.kth.se/~artho/papers/tap-2013.pdf)).

**Review by carlocab — APPROVED** [review]
Here's to another SAT solver.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**